utils/slirp4netns/Makefile

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (C) 2006-2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
Remove it or assign it to yourself according to CONTRIBUTING.md in this repository.